actionview/lib/action_view/helpers/text_helper.rb

```diff
@@ -206,6 +206,11 @@ def excerpt(text, phrase, options = {})
       # +plural+ is supplied, it will use that when count is > 1, otherwise
       # it will use the Inflector to determine the plural form.
       #
+      # If passed an optional +locale+ parameter, the word will be pluralized
```

Comment:
Done in https://github.com/noniq/rails/commit/aed3ac4d9ab058a88d4dc7f30a028222e68d1425 :-)